labgrid/util/agentwrapper.py

```diff
@@ -103,12 +103,12 @@ def call(self, method, *args, **kwargs):
 
         raise AgentError(f"unknown response from agent: {response}")
 
-    def load(self, name):
+    def load(self, name, path=None):
         if name in self.loaded:
             return self.loaded[name]
 
         filename = os.path.join(
-            os.path.abspath(os.path.dirname(__file__)),
+            path if path else os.path.abspath(os.path.dirname(__file__)),
             'agents', f'{name}.py')
```

Comment:
Why do you prefer to point to a directory containing the `agents` subdirectory, instead of directly to the directory containing `{name}.py`? I think it would be less confusing.

Reply:
I agree; that would be less confusing.  Not sure why I didn't choose that first.